modular_skyrat/modules/company_imports/code/datums/cargo_pack.dm

```diff
@@ -1,6 +1,8 @@
 #define CARGO_CUT 0.05
 
 /datum/supply_pack/armament
+	goody = TRUE
+	crate_type = /obj/structure/closet/crate/large/import
```

Comment:
Won't this cause crates that are worth money to be sent through goodies?

Reply:
Items only come as crates once more than five of them are ordered, and a fee for the crate is automatically charged to the buyer's account.